File: basics/Q40/main.go
package main

import "fmt"

func main() {
	var (
		a int
		b int
	)
	fmt.Print("Enter a: ")
	fmt.Scan(&a)
	fmt.Print("Enter b: ")
	fmt.Scan(&b)
	fmt.Printf("a + b = %d\n", a+b)
	fmt.Printf("a - b = %d\n", a-b)
	fmt.Printf("a * b = %d\n", a*b)
	fmt.Printf("a / b = %d\n", a/b)
	fmt.Printf("a^2 + b^2 = %d\n", a*a+b*b)
	fmt.Printf("a^3 + b^3 = %d", a*a*a+b*b*b)
}

File: basics/Q40/main.rs
use std::io;

fn main() {
    let mut input = String::new();

    println!("Enter a:");
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let a: i64 = input.trim().parse().expect("Not a valid number");

    input.clear();

    println!("Enter b:");
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let b: i64 = input.trim().parse().expect("Not a valid number");

    println!("a + b = {}", a + b);
    println!("a - b = {}", a - b);
    println!("a * b = {}", a * b);

    if b == 0 {
        eprintln!("a / b = division by zero");
    } else {
        println!("a / b = {}", a / b);
    }

    println!("a^2 + b^2 = {}", a * a + b * b);
    println!("a^3 + b^3 = {}", a * a * a + b * b * b);
}